src/components/Input/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { Component, InputHTMLAttributes } from "react";
+import { IconType } from "react-icons/lib/esm/iconBase";
+
+type Props = {
+    inputType: InputHTMLAttributes<HTMLInputElement>["type"];
+    placeholder: InputHTMLAttributes<HTMLInputElement>["placeholder"];
```

Comment:
The placeholder can simply be of a `string` type. I think this is what this converges to anyway